### DISCUSSION TITLE
Custom DNS Resolver

### DISCUSSION THREAD

**Discussion by zQueal** [minbrowser/min#2363] [Ideas]
I run my own DNS server which allows me to basically compile 1 hosts file and take it with me on all my devices. Min is missing the ability to use custom DNS resolvers.

Would be nice to have this feature.

**Comment by PalmerAL** [comment]
Are you not able to change your DNS settings at the OS level?

**Reply by zQueal** [reply]
While it's possible, it's disadvantageous versus application based DNS settings. For example, using system wide custom DNS blocks windows updates and other telemetry (virus definitions, etc) which are essential. Using allowing the application itself to set custom a custom DNS resolver sidesteps this issue as only requests which originate from Min will use the custom resolver.
